hips/tiles/tile.py

```diff
@@ -0,0 +1,131 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from astropy.utils.data import get_pkg_data_filename
```

Comment:
The `HipsDescription` constructor only expects the filename, but in order to get that, `get_pkg_data_filename` is required. This is the line that does it:

```
filename = get_pkg_data_filename('data/properties.txt')
hips_description = HipsDescription.read(filename)

```
This is how it was done in [test_description.py](https://github.com/hipspy/hips/blob/master/hips/tiles/tests/test_description.py) as well. I tried running the test by only passing the filename as a string, but the test case fails, giving this error:

> FileNotFoundError: [Errno 2] No such file or directory: 'data/properties.txt'